### PR TITLE
Phpstan

### PR DESCRIPTION
**Pull request by greg0ire**
I am targeting this branch, because some fixes do not apply on stable. Call me lazy, but since we are releasing soonish, I think splitting is not worth the hassle.

## Changelog

```markdown
### Fixed
- various bugs found with phpstan
```

### CODE REVIEW

File: Invalidation/DoctrineORMListenerContainerAware.php [hunk 1]
```diff
@@ -21,6 +21,7 @@ class DoctrineORMListenerContainerAware implements EventSubscriber
     protected $listener;
 
     protected $service;
+    protected $container;
```
Comment (core23) [inline Invalidation/DoctrineORMListenerContainerAware.php]: Please add a PHPDoc

File: Invalidation/DoctrinePHPCRODMListenerContainerAware.php [hunk 2]
```diff
@@ -21,6 +21,7 @@ class DoctrinePHPCRODMListenerContainerAware implements EventSubscriber
     protected $listener;
 
     protected $service;
+    protected $container;
```
Comment (core23) [inline Invalidation/DoctrinePHPCRODMListenerContainerAware.php]: PHPDoc missing

File: Adapter/SymfonyCache.php [hunk 3]
```diff
@@ -62,6 +62,11 @@ class SymfonyCache implements CacheAdapterInterface
     protected $timeouts;
 
     /**
+     * @var Filesystem
+     */
+    protected $filesystem;
+
+    /**
      * Constructor.
```
Comment (OskarStark) [inline Adapter/SymfonyCache.php]: can you remove this comment please?

File: Adapter/SymfonyCache.php [hunk 4]
```diff
@@ -141,7 +146,10 @@ public function flush(array $keys = ['all']): bool
                 if ($result) {
                     $result = substr($content, -2) == 'ok';
                 } else {
-                    return $content;
+                    throw new \UnexpectedValueException(sprintf(
```
Comment (OskarStark) [inline Adapter/SymfonyCache.php]: do we need a linebreak here?

File: Adapter/VarnishCache.php [hunk 5]
```diff
@@ -154,6 +154,11 @@ public function cacheAction(Request $request)
         $subRequest = Request::create('', 'get', $parameters, $request->cookies->all(), [], $request->server->all());
 
         $controller = $this->resolver->getController($subRequest);
+        if (!$controller) {
+            throw new \UnexpectedValueException(
```
Comment (OskarStark) [inline Adapter/VarnishCache.php]: do we need a linebreak here?

File: Adapter/SymfonyCache.php (hunk 4 above)
Comment (Soullivaneuh) [reply Adapter/SymfonyCache.php]: `sprintf` may be avoid here. We should discuss about it on dev-kit.

File: Invalidation/DoctrineORMListenerContainerAware.php [hunk 6]
```diff
@@ -23,6 +23,11 @@ class DoctrineORMListenerContainerAware implements EventSubscriber
     protected $service;
 
     /**
+     * @param ContainerInterface
+     */
+    protected $container;
```
Comment (Soullivaneuh) [inline Invalidation/DoctrineORMListenerContainerAware.php]: Eeeeeeeeeeeeerrrrrkkkk!!! :smile:

**Pull request merged**